crates/iceberg/src/spec/snapshot.rs

```diff
@@ -192,13 +191,6 @@ impl Snapshot {
             partition_type_provider,
         )
     }
-
-    pub(crate) fn log(&self) -> SnapshotLog {
```

Comment:
Yes, it's because deprecated function `append_snapshot` is the only function that calls it. Since we are reverting that change, I will revert this as well